Documentation/ceph-object-multisite-crd.md

```diff
@@ -20,6 +20,9 @@ kind: CephObjectRealm
 metadata:
   name: realm-a
   namespace: rook-ceph
+spec:
+#  pull:
+#    endpoint: http://10.2.105.133:80
```

Comment:
In the doc how about uncommenting it for readability? Obviously they need to change the endpoint before using it in their cluster though. Maybe add a comment above the endpoint as well.